src/sentry/tasks/embeddings_grouping/utils.py

```diff
@@ -358,33 +359,63 @@ def get_events_from_nodestore(
     )
 
 
+def _make_seer_call(
+    create_grouping_records_request: CreateGroupingRecordsRequest, project_id: int
+) -> BulkCreateGroupingRecordsResponse | None:
+    try:
+        seer_response = _retry_operation(
+            post_bulk_grouping_records,
+            create_grouping_records_request,
+            retries=3,
+            delay=2,
+            exceptions=ServiceUnavailable,
+        )
+    except ServiceUnavailable:
+        extra = {
+            "project_id": project_id,
+            "error": "Service Unavailable",
+        }
+        logger.exception(
+            "tasks.backfill_seer_grouping_records.seer_service_unavailable",
+            extra=extra,
+        )
```

Comment:
Nit: Is there are reason to include `service unavailable` in the tags when it's in the log event name?

Reply:
nope, I can remove it